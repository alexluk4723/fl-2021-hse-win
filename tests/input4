hi:= ^1234235^
return hi
