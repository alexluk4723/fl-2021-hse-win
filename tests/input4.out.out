graph graphname {
hi [label="head:hi"]
hi -- hi_1
hi_1 [label="1234235"]
MAIN__ [label="MAIN" color=red]
MAIN__ -- hi
}
